.github/workflows/ci.yml

```diff
@@ -69,7 +69,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        GOOS: ['linux', 'windows', 'darwin']
+        GOOS: ['linux', 'darwin']
```

Comment:
We need to ensure the snapshooter can be compiled on windows platform, so that it can be imported by other components (buildkit/nerdctl).

Reply:
Windows compilation is back